tests/Unit/Domain/Creators/Test_RotatedIntervals.cpp

```diff
@@ -109,15 +111,17 @@ void test_rotated_intervals_factory() {
   INFO("Rotated intervals factory");
   const OrientationMap<1> flipped{
       std::array<Direction<1>, 1>{{Direction<1>::lower_xi()}}};
-  const auto domain_creator =
-      TestHelpers::test_factory_creation<DomainCreator<1>>(
-          "RotatedIntervals:\n"
-          "  LowerBound: [0.0]\n"
-          "  Midpoint:   [0.5]\n"
-          "  UpperBound: [1.0]\n"
-          "  IsPeriodicIn: [True]\n"
-          "  InitialGridPoints: [[3,2]]\n"
-          "  InitialRefinement: [2]\n");
+  const auto domain_creator = TestHelpers::test_factory_creation<
+      DomainCreator<1>, domain::OptionTags::DomainCreator<1>,
+      TestHelpers::domain::BoundaryConditions::
+          MetavariablesWithoutBoundaryConditions<1>>(
+      "RotatedIntervals:\n"
+      "  LowerBound: [0.0]\n"
+      "  Midpoint:   [0.5]\n"
+      "  UpperBound: [1.0]\n"
+      "  IsPeriodicIn: [True]\n"
+      "  InitialGridPoints: [[3,2]]\n"
+      "  InitialRefinement: [2]\n");
```

Comment:
This also looks like it belongs in the next PR

Reply:
No, factory creation needs to be able to create all domain creators, and `Interval` no longer supports `void` as a metavariable